optimum/intel/openvino/__init__.py

```diff
@@ -78,6 +71,13 @@
 )
 from .modeling_seq2seq import OVModelForPix2Struct, OVModelForSeq2SeqLM, OVModelForSpeechSeq2Seq, OVModelForVision2Seq
 from .modeling_visual_language import OVModelForVisualCausalLM
+from .quantization import (
```

Comment:
does it works if nncf is not installed?
Should not be under like it was imported before?

if is_nncf_available():

Reply:
looks like yes, it works, because modeling tests are passed in precommit (this test scope explicitly uninstall nncf)